ui/semantic.json

```diff
@@ -0,0 +1,22 @@
+{
+  "base": "semantic/",
```

Comment:
BEGIN IGNORE PULL REQUEST. (look for the next comment about CONTINUING THE PULL REQUEST). 

NOTE: Semantic UI is a user interface library for rich controls -> Allowed me to do VERY QUICK and attractive UI. The reason this folder exists here in the UI project is in case we want to change the THEME we're using. Including the Semantic UI directory in the project, we can rebuild the CSS with whatever theme we want. The only files that get included in the UI build (and then the `statik.go`) is the CSS file and any other dependent artifacts (fonts and icons). You can rebuild the Semantic theme after changes by running a tool called `Gulp`. However, If you just leave it alone, the current UI theme is fine. One last point? I went this direction so all resources would be in the executable. If we'd pointed at CSS on a CDN (like many folks do with Bootstrap), then if we install the app in a Docker container or other it would need internet access just for the css/fonts/icons. Doing it this way means that all artifacts are in the executable and would be browsable whether or not you have internet access.